src/qt/optionsdialog.cpp

```diff
@@ -148,6 +148,10 @@ void OptionsDialog::setModel(OptionsModel *_model)
         if (_model->isRestartRequired())
             showRestartWarning(true);
 
+        // Prune values are in GB to be consistent with intro.cpp
+        static constexpr uint64_t nMinDiskSpace = (MIN_DISK_SPACE_FOR_BLOCK_FILES / GB_BYTES) + (MIN_DISK_SPACE_FOR_BLOCK_FILES % GB_BYTES) ? 1 : 0;
+        ui->pruneSize->setRange(nMinDiskSpace, std::numeric_limits<int>::max());
```

Comment:
Why was this changed in the first place?

Couldn't you restore the `ui->pruneSize->setMinimum(nMinDiskSpace);` like it was before?

Reply:
The default max is 99.

Reply:
ACK, doc is here: https://doc.qt.io/qt-5/qspinbox.html#maximum-prop.